pkg/reconciler/instances/ory/hydra/hydra.go

```diff
@@ -2,12 +2,12 @@ package hydra
 
```

Comment:
How about renaming this file and folder to syncer or hydraSyncer?

Reply:
Yes, good proposal, I've renamed it